src/images/icons/about-background-line-desktop.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 1437 618" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
```suggestion
<svg preserveAspectRatio="none" viewBox="0 0 1437 618" fill="none" xmlns="http://www.w3.org/2000/svg">
```

If we want the background to stretch all the way when the height doesn't allow it (e.g.: on `big-desktop`s), consider adding this. I think it's fine to stretch these graphics.